### PR TITLE
add oxidized package and module

### PR DESCRIPTION
**Pull request by WilliButz**
The package is not supposed to be upstreamed yet.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by fpletz — APPROVED** [review]

File: nixos/modules/services/admin/oxidized/default.nix [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      type = types.path;
+      default = "/var/lib/oxidized";
+      description = "State directory for the oxidized service.";
+    };
+
+    configFile = mkOption {
+      type = types.path;
+      description = ''
+        Path to the oxidized configuration file. This defaults to
+        the output path of the <option>configuration</option> option.
```
Comment (globin) [inline nixos/modules/services/admin/oxidized/default.nix]: Docs don't fit the behaviour

File: nixos/modules/services/admin/oxidized/default.nix [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      '';
+    };
+
+    routerDB = mkOption {
+      type = types.path;
+      description = ''
```
Comment (globin) [inline nixos/modules/services/admin/oxidized/default.nix]: add a default?

**Review by globin — DISMISSED** [review]
Nitpicks

**Review by fpletz — DISMISSED** [review]
Also remove the WIP from the first commit.

File: nixos/modules/services/admin/oxidized/default.nix (hunk 2 above)
Comment (WilliButz) [reply nixos/modules/services/admin/oxidized/default.nix]: I added an example instead

**Pull request merged**